app/views/focusingInput.js

```diff
@@ -22,7 +22,7 @@ module.exports = component('FocusingInput', {
     value,
     ref,
     onChange,
-    onKeyUp: (event) => {
+    onKeyUp (event) {
```

Comment:
Again, I don't think that we should introduce simplified syntax for functions *in case they are part of of objects*.